src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/CollectionViewUITests.CollectionViewItemsUpdatingScrollMode.cs

```diff
@@ -46,10 +46,9 @@ public void KeepItemsInView()
 		// KeepScrollOffset (src\Compatibility\ControlGallery\src\Issues.Shared\CollectionViewItemsUpdatingScrollMode.cs)
 		[Test]
 		[Category(UITestCategories.CollectionView)]
+		[FailsOnAllPlatforms]
```

Comment:
What is the difference between Ignore that I have seen on other tests and FailsOnAllPlatforms?

Reply:
Internally, no differences. However, it can cause confusion so I have made changes to always follow the same pattern using FailsOn with a description.